packages/backend-auth/API.md

```diff
@@ -26,6 +26,12 @@ export type AmazonProviderFactoryProps = Omit<AmazonProviderProps, 'clientId' |
     clientSecret: BackendSecret;
 };
 
+// @public (undocumented)
+export type AmplifyAuthFactoryProps = Omit<AuthProps, 'outputStorageStrategy' | 'loginWith'> & TriggerConfig & {

```

Comment:
this type already exists here https://github.com/aws-amplify/amplify-backend/blob/7398baa80461dee7d3abd47b51926c640fb6b6e8/packages/backend-auth/src/factory.ts#L20-L21